addons/l10n_es/data/mod303.xml

```diff
@@ -0,0 +1,967 @@
+                                <field name="code">aeat_mod_303_title_7</field>
+                                <field name="children_ids">
+                                    <record id="mod_303_casilla_25" model="account.report.line">
+                                        <field name="name">[25] Base taxable</field>
+                                        <field name="code">aeat_mod_303_25</field>
+                                        <field name="groupby">account_id</field>
+                                        <field name="foldable" eval="True"/>
+                                        <field name="expression_ids">
+                                            <record id="mod_303_casilla_25_balance" model="account.report.expression">
+                                                <field name="label">balance</field>
+                                                <field name="engine">tax_tags</field>
+                                                <field name="formula">mod_303_25</field>
+                                            </record>
+                                        </field>
+                                    </record>
+                                    <record id="mod_303_casilla_26" model="account.report.line">
+                                        <field name="name">[26] Contributions</field>
+                                        <field name="code">aeat_mod_303_26</field>
+                                        <field name="groupby">account_id</field>
+                                        <field name="foldable" eval="True"/>
+                                        <field name="expression_ids">
+                                            <record id="mod_303_casilla_26_balance" model="account.report.expression">
+                                                <field name="label">balance</field>
+                                                <field name="engine">tax_tags</field>
+                                                <field name="formula">mod_303_26</field>
+                                            </record>
+                                        </field>
+                                    </record>
+                                </field>
+                            </record>
+                            <record id="mod_303_casilla_27" model="account.report.line">
+                                <field name="name">[27] Total accrual contributions</field>
+                                <field name="code">aeat_mod_303_27</field>
+                                <field name="aggregation_formula">aeat_mod_303_03.balance + aeat_mod_303_155.balance + aeat_mod_303_06.balance + aeat_mod_303_09.balance + aeat_mod_303_11.balance + aeat_mod_303_13.balance + aeat_mod_303_15.balance + aeat_mod_303_18.balance + aeat_mod_303_21.balance + aeat_mod_303_24.balance + aeat_mod_303_26.balance</field>
```

Comment:
```suggestion
                                <field name="aggregation_formula">aeat_mod_303_152.balance + aeat_mod_303_03.balance + aeat_mod_303_155.balance + aeat_mod_303_06.balance + aeat_mod_303_09.balance + aeat_mod_303_11.balance + aeat_mod_303_13.balance + aeat_mod_303_15.balance + aeat_mod_303_158.balance + aeat_mod_303_18.balance + aeat_mod_303_21.balance + aeat_mod_303_24.balance + aeat_mod_303_26.balance</field>
```
This was added by https://github.com/odoo/enterprise/issues/39477

Reply:
fixed